mail/postfix/src/opnsense/mvc/app/controllers/OPNsense/Postfix/forms/general.xml

```diff
@@ -29,6 +29,12 @@
         <type>text</type>
         <help>The 'Listen IPs' parameter specifies the IP address to listen to. Default is to listen on all interfaces.</help>
     </field>
+    <field>
+        <id>general.inet_port</id>
+        <label>Listen Port</label>
+        <type>text</type>
+        <help>Port to listen on.</help>
```

Comment:
Would add `TCP ` before the text and I would add information about TCP/25 to be the default port and other servers must reach the MX on TCP/25.